src/backend/distributed/deparser/ruleutils_16.c

```diff
@@ -0,0 +1,9193 @@
+/*-------------------------------------------------------------------------
```

Comment:
is there anything interesting to check in ruleutils? It is a non-trivial review :)